pycbc/inference/models/hierarchical.py

```diff
@@ -723,24 +733,43 @@ def total_loglr(self):
             # not using self.primary_model.current_params, because others_model
             # may have its own static parameters
             current_params_other = other_model.current_params.copy()
-            for i in range(nums):
+            if not self.accelerate_loglr:
```

Comment:
Can you choose a more descriptive name to what the option does (e.g. *how* does it make the likelihood faster than than just the fact that it does)

Reply:
Is `self.static_margin_params_in_other_models` better?

Reply:
@ahnitz In this PR, we don't apply any amplitude and phase correction, so I will not include these in the option name.